File: orchestrator/src/main/java/br/com/orchestrator/application/dto/Adress.java
package br.com.orchestrator.application.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Adress {
    private Integer id;
    private String code;
    private String street;
    private String number;
    private String district;
}

File: user-registration/src/main/kotlin/br/com/userregistration/application/dto/Adress.kt
package br.com.userregistration.application.dto

class Adress {
    private val code: String? = null
    private val street: String? = null
    private val number: String? = null
    private val district: String? = null

    fun getCode(): String? {return this.code};
    fun getStreet(): String? {return this.street};
    fun getNumber(): String? {return this.number};
    fun getDistrict(): String? {return this.district};

    override fun toString(): String {
        return "Adress(code=$code, street=$street, number=$number, district=$district)"
    }
}
